docs/deployments/deploying-behind-a-proxy.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Deploying a Clerk app behind a proxy
+description: Learn how to deploy a Clerk app to a web server using proxies and reverse proxies like nginx or Caddy, or using Docker
+---
+
+When deploying a Clerk app beind a proxy, you may need to proxy two headers for Clerk to work correctly. The two headers that you must forward are:
+
+- `X-Forwarded-Host`
+- `X-Forwarded-Proto`
+
+## `nginx` configuration example
+
+The following can be added to your `nginx` configuration to forward these two headers to your application.
+
+```nginx
+proxy_set_header X-Forwarded-Host $host;
+proxy_set_header X-Forwarded-Proto https;
+```
+
+## How to test if the headers are forwarded correctly:
+
+You can console log the `X-forwarded-host` and `X-forwarded-proto` to check what the value is. The following example is for Next.js.
```

Comment:
What should the values be, so that the user knows that the forwarding worked?

Reply:
- `X-Forwarded-Host`: The original host that the client requested, like if your public domain is `example.com`, then `X-Forwarded-Host` should appear as `example.com`

- `X-Forwarded-Proto`: The original protocol used by the client. If your site is served over HTTPS, then `X-Forwarded-Proto` should appear as https... Clerk requires HTTPS so it needs to be HTTPS

Reply:
Example expected output from the console log, assuming that the application's production domain is `clerk.com` below. I'll add this to the PR now.

```
=========================
Request Headers
x-forwarded-host clerk.com
x-forwarded-proto https
=========================
```

Reply:
Added in https://github.com/clerk/clerk-docs/pull/1813/commits/ed19a2dea296c2ce6f42d98684070813d3289bc0